pkg/volume/gce_pd/gce_util.go

```diff
@@ -146,7 +146,7 @@ func (gceutil *GCEDiskUtil) CreateVolume(c *gcePersistentDiskProvisioner) (volum
 		return "", 0, err
 	}
 
-	err = cloud.CreateDisk(name, zone.FailureDomain, int64(requestGB))
+	err = cloud.CreateDisk(name, zone.FailureDomain, int64(requestGB), *c.options.CloudTags)
```

Comment:
For `name`, what if we include the PV name?  You've already created a gce-specific name here:  https://github.com/kubernetes/kubernetes/blob/master/pkg/volume/gce_pd/gce_pd.go#L390

Instead of another one "kube-dynamic-UID", we can use the unique name generated on the PV itself.

This obviates the need to put pv.Name in the description because both volumes are named exactly the same.